IDE/MSVS-2019-AZSPHERE/user_settings.h

```diff
@@ -0,0 +1,74 @@
+#define SERVER_IP "192.168.1.123" 
```

Comment:
Move these top two into the `_USER_SETTINGS_H_` section. Add wolfSSL license header.